### PR TITLE
translate "ts for js programmers" in french

### PR DESCRIPTION
**Pull request by mk360**

### CODE REVIEW

**Comment by mk360** [comment]
Hello @ManuSquall @Ascor8522 
It's been a while since this PR was open, could you take a look at it so it either gets merged or improved upon?

**Comment by ManuSquall** [comment]
Hello @mk360 ,just saw it, thanks for tagging us, will revie it asap
Thanks you for this PR 🥳

File: docs/documentation/fr/get-started/TS for JS Programmers.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
+---
+title: TypeScript pour les développeurs JavaScript
+short: TypeScript pour les développeurs JS
+layout: docs
+permalink: /fr/docs/handbook/typescript-in-5-minutes.html
+oneline: Apprenez comment TypeScript étend JavaScript
+---
+
+TypeScript a une relation inhabituelle avec JavaScript. TypeScript offre toutes les fonctionnalités de JavaScript, avec une couche supplémentaire de fonctionnalités : le système de types.
```
Comment (ManuSquall) [inline docs/documentation/fr/get-started/TS for JS Programmers.md:9]: "avec une couche supplémentaire de fonctionnalités : le système de *typage*" conviendrait également, mais bon c'est subjectif.

File: docs/documentation/fr/get-started/TS for JS Programmers.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+
+JavaScript fournit des primitives, comme `string` et `number`, mais aucune vérification n'est faite pour s'assurer que les assignations que vous faites sont correctes. TypeScript le fait.
+
+Cela signifie que votre code JavaScript existant est également du code TypeScript. L'avantage principal de TypeScript est sa capacité à exposer les comportements imprévus dans votre code, diminuant la chance de trouver des bugs.
```
Comment (ManuSquall) [inline docs/documentation/fr/get-started/TS for JS Programmers.md:13]: "les comportements imprévus dans votre code, diminuant *les risques de bugs.*" ou "code, diminuant *les risques d'avoir des bugs.*" me parait mieux.

File: docs/documentation/fr/get-started/TS for JS Programmers.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+Par exemple, en créant une variable et en lui assignant une certaine valeur, TypeScript va utiliser cette valeur en tant que type.
+
+```ts twoslash
+let helloWorld = "Hello World";
+//  ^?
+```
+
+En comprenant comment JavaScript fonctionne, TypeScript peut créer un système qui accepte du code JavaScript, avec des types. Le résultat est un système de types qui n'a pas besoin de déclarations explicites de types dans votre code. C'est comme ça que TypeScript sait que `helloWorld` est un `string` dans l'exemple précédent (TypeScript a _inféré_ le type `string` de `helloWorld`).
+
+Il se peut que vous ayez écrit du JavaScript dans Visual Studio Code, et ayez obtenu de l'autocomplétion de la part de l'éditeur. Visual Studio Code utilise TypeScript dans le capot pour faciliter le travail avec JavaScript.
```
Comment (ManuSquall) [inline docs/documentation/fr/get-started/TS for JS Programmers.md:29]: "Visual Studio Code utilise TypeScript *en arrière plan* pour faciliter le travail avec JavaScript" seems more common in french than "dans le capot" for me :-)

**Review by ManuSquall — CHANGES_REQUESTED** [review]
Thanks a lot for your PR !
Indexing mainly the "dans le capot" expression but have done some suggestions / comments in any case.

**Comment by mk360** [comment]
All done

**Review by ManuSquall — APPROVED** [review]
LGTM

**Comment by ManuSquall** [comment]
Thanks for this PR 🥳🥳

**Comment by ManuSquall** [comment]
LGTM

**Pull request merged**